pkgs/applications/networking/instant-messengers/gtkcord/default.nix

```diff
@@ -0,0 +1,45 @@
+{ makeDesktopItem, lib, buildGoModule, fetchFromGitHub, pkgconfig, go, gnome3, libhandy }:
+
+buildGoModule rec {
+  pname = "gtkcord";
+  version = "0.0.4";
+
+  src = fetchFromGitHub {
+    owner = "diamondburned";
+    repo = "gtkcord3";
+    rev = "v${version}";
+    sha256 = "0n9600s50zpx8kv89icrlxarm3633gnizg1sq1zpask8bbv2xnfd";
+  };
+
+  nativeBuildInputs = [ pkgconfig go ];
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.